x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_started_session_details.test.tsx

```diff
@@ -0,0 +1,397 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import { mountWithIntl, shallowWithIntl } from 'test_utils/enzyme_helpers';
+import { BrowserFields } from 'x-pack/plugins/secops/public/containers/source';
+
+import { mockBrowserFields } from '../../../../containers/source/mock';
+import { mockEcsData, TestProviders } from '../../../../mock';
+
+import {
+  AuditdStartedSessionDetails,
+  AuditdStartedSessionLine,
+} from './auditd_started_session_details';
+
+describe('AuditdStartedSessionDetails', () => {
+  describe('rendering', () => {
+    test('it renders the default AuditdStartedSessionDetails', () => {
+      // I cannot and do not want to use the mocks for the snapshot tests as they are too heavy
```

Comment:
consider adding `BrowserFields` as the subject of this comment, as in [PR 33464](https://github.com/elastic/kibana/pull/33464/files#diff-4c683abaf166ef2c1f3c8e78af908498R29)